### DISCUSSION TITLE
is it working with "spatie/laravel-data"

### DISCUSSION THREAD

**Discussion by sandip-b-7span** [dedoc/scramble#248] [Q&A] [answered]
Hello Guys,
Anyone has tried this package with "spatie/laravel-data"? I think this is not working with the same so if anyone find any solution then please share.

Thanks.

**Comment by jeffreyvanhees** [comment, marked as answer]
See https://github.com/dedoc/scramble/discussions/168

**Reply by sandip-b-7span** [reply]
@jeffreyvanhees  so as per your reference the package does not support spatie/laravel-data right?

**Reply by jeffreyvanhees** [reply]
Not yet, but maybe in the nearby feature :)

**Reply by sandip-b-7span** [reply]
@jeffreyvanhees  What if i want to display custom name instead of route name in the list of endpoints. is it possible in this package?
![image](https://github.com/dedoc/scramble/assets/115146134/76467f8a-54cf-4c6f-8b44-a7f373fccd9c)

**Reply by romalytvynenko** [reply]
@sandip-b-7span please check docs: https://scramble.dedoc.co/usage/request#adding-title-and-description
